components/authn-service/cmd/authn-service/serve.go

```diff
@@ -109,6 +114,27 @@ func serve(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("missing address for authz service")
 	}
 
+	if c.CerealAddress == "" {
+		return fmt.Errorf("missing address for cereal service")
+	}
+
+	factory := secureconn.NewFactory(*serviceCerts, secureconn.WithVersionInfo(
```

Comment:
this probably isn't a huge deal, but you already have one of these factories in server.go

Reply:
Yeah I wasn't sure what the cleanest way of dealing with that was.